### PR TITLE
[17.0][MIG] hr_timesheet_report_invoice

### PR DESCRIPTION
**Pull request by Anxo82**
Standard migration.

### CODE REVIEW

**Comment by Anxo82** [comment]
@dalonsod Review please.
Thanks.

**Comment by dalonsod** [comment]
Could you provide a screenshoot for v17 with this addon working? I couldn't obtain a Timsheets Report with an invoice number in the header

**Comment by Anxo82** [comment]
imagen


imagen

**Comment by dalonsod** [comment]
Ok, I'll check it ASAP again

File: hr_timesheet_report_invoice/report/account_analytic_line_inherit.xml [hunk 1]
```diff
@@ -1,10 +1,9 @@
-<?xml version="1.0" encoding="utf-8"?>
 <odoo>
     <template id="account_analytic_line_inherit" inherit_id="hr_timesheet.report_timesheet">
         <xpath expr="//t[@t-set='show_project']" position="after">
             <t
                 t-set="invoice_name"
-                t-value="docs.sudo().get_invoice_name()"
+                t-value="docs and docs[0].sudo().get_invoice_name()"
```
Comment (dalonsod) [inline hr_timesheet_report_invoice/report/account_analytic_line_inherit.xml]: And why this change?

Comment (Anxo82) [reply hr_timesheet_report_invoice/report/account_analytic_line_inherit.xml]: @dalonsod  I made this change to check that the document is not empty; I believe there was an _IndexError_

Comment (dalonsod) [reply hr_timesheet_report_invoice/report/account_analytic_line_inherit.xml]: But in that case this should be enough:
```suggestion
                t-value="docs and docs.sudo().get_invoice_name()"
```
and we keep multiple `docs` selection. Notice that `get_invoice_name()` is specifically designed to be called with a set of timesheets, passing only one breaks the functionality (see definition).

Comment (dalonsod) [reply hr_timesheet_report_invoice/report/account_analytic_line_inherit.xml]: Review again, making a rollback in this line (keeping v15 original code) and still works, so no changes for this line should be made in v17 IMO, please remove it

**Comment by Anxo82** [comment]
@dalonsod Done! when you give its OK, squash commits.
Thanks

**Comment by dalonsod** [comment]
:+1: you can squash

**Review by dalonsod — APPROVED** [review]
Let's merge this!
